pisek/config/config_types.py

```diff
@@ -0,0 +1,48 @@
+# pisek  - Tool for developing tasks for programming competitions.
+#
+# Copyright (c)   2023        Daniel Skýpala <daniel@honza.info>
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# any later version.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>
+
+from enum import StrEnum, auto
+
+
+class TaskType(StrEnum):
+    batch = auto()
+    communication = auto()
+
+
+class JudgeType(StrEnum):
+    diff = auto()
+    judge = auto()
+
+
+class Scoring(StrEnum):
+    equal = auto()
+    min = auto()
```

Comment:
These names aren't very intuitive.

Reply:
What do you suggest?

Reply:
I don't know. I feel like `all` was a bit more intuitive, but I'm not sure it that's that great either.